tests/test_connection.py

```diff
@@ -33,7 +33,7 @@ def con(fshost, loop):
 @pytest.mark.parametrize(
     'password, expect_auth',
     [('doggy', False), ('ClueCon', True)],
-    ids=lambda item: "pw={}, expect_auth={}".format(*item),
+    # ids=lambda item: "pw={}, expect_auth={}".format(*item),
```

Comment:
Is there a reason this was removed?

It's not a big deal but iirc these `ids` tags sure are handy when doing manual runs of the test suite from console.

Reply:
I have added it back in, but it works differently now.